src/OpenFeature/Hooks/TracingHook.cs

```diff
@@ -0,0 +1,54 @@
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Threading;
+using System.Threading.Tasks;
+using OpenFeature.Model;
+
+namespace OpenFeature.Hooks;
+
+/// <summary>
+/// Stub.
```

Comment:
This doc doesn't appear correct.

Reply:
Fixed with: https://github.com/open-feature/dotnet-sdk/pull/338/commits/345a1648714e8fb4eaea986082f47d8ab91ff0c1